### DISCUSSION TITLE
problem with Livewire Breeze Logout and back button still shows dashboard

### DISCUSSION THREAD

**Discussion by ZeeshanX4** [livewire/livewire#8595] [1. Help]
Hi all,

In downloaded a fresh copy of Laravel 11 and installed Breeze, which gives option of Livewire (Volt Class API). the wire:navigate and navigate: true are all working fine.

When I sign out, and it brings us welcome page again, and then if I press back button, it shows the (authenticated) dashboard again.

**Comment by joshhanley** [comment]
@mzab1985 I would suggest opening an issue on the Breeze repo then.
